README.md

```diff
@@ -168,8 +168,7 @@ In Windows, the script is generally installed in
     C:\>\Python34\python \Python34\Scripts\mypy PROGRAM
 
 If you are on Windows using Python 3.3 or 3.4, and would like to use XML reports
```

Comment:
If #4007 gets merged before this, this sentence can be removed, as there are wheels for all Python versions on Windows now, and installing via test-requirements will "just work".